content/en/agent/kubernetes/operator_configuration.md

```diff
@@ -0,0 +1,214 @@
+| agent.config.securityContext.capabilities.add                                                              | Added capabilities                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| agent.config.securityContext.capabilities.drop                                                             | Removed capabilities                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| agent.config.securityContext.privileged                                                                    | Run container in privileged mode. Processes in privileged containers are essentially equivalent to root on the host. Defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| agent.config.securityContext.procMount                                                                     | procMount denotes the type of proc mount to use for the containers. The default is DefaultProcMount which uses the container runtime defaults for readonly paths and masked paths. This requires the ProcMountType feature flag to be enabled.                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
I just realized that many of these are not in code ticks, so `procMount`, `DefaultProcMount`, etc, through this whole table